webpack/webpack.common.js

```diff
@@ -68,9 +68,11 @@ module.exports = (options) => ({
             }
         }),
         new CopyWebpackPlugin([
-            { from: './node_modules/swagger-ui/dist/css', to: 'swagger-ui/dist/css' },
-            { from: './node_modules/swagger-ui/dist/lib', to: 'swagger-ui/dist/lib' },
-            { from: './node_modules/swagger-ui/dist/swagger-ui.min.js', to: 'swagger-ui/dist/swagger-ui.min.js' },
+            //{ from: './node_modules/swagger-ui/dist/css', to: 'swagger-ui/dist/css' },
```

Comment:
Are these local settings for Ioannis? I don't know much about this webpack but it could be local settings for you to run it, maybe we shouldn't commit this to develop